lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -410,6 +425,7 @@ public static function get_merged_data( $settings = array(), $origin = 'user' )
 		$theme_support_data = WP_Theme_JSON_Gutenberg::get_from_editor_settings( $settings );
 
 		$result = new WP_Theme_JSON_Gutenberg();
+		$result->merge( self::get_block_data() );
 		$result->merge( self::get_core_data() );
```

Comment:
Would this make more sense?

```suggestion
		$result->merge( self::get_core_data() );
		$result->merge( self::get_block_data() );
```

Reply:
In determining the best ordering here we should consider the current loading order of stylesheets. If we start moving everyone to block styles and they start getting added to the global styles _before_ the Core `theme.json` then we could get anomalies.

Reply:
I've switched the two, there are no conflicts as of today and blocks_data after core_data seems to have more sense.